universal_doc_importer/github/connectors.py

```diff
@@ -28,7 +30,7 @@ def _parse_url(self, url: str) -> str:
 
     def get_repo_map(self, url, extensions):
         repo_name = self._parse_url(url=url)
-        folder_name = repo_name.split('/')[1]
+        folder_name = repo_name.split('/')[0] + '-' + repo_name.split('/')[1]
```

Comment:
`'-'.join(repo_name.split('/')[:2])`

also add a comment in the code with an example, please